Makefile

```diff
@@ -49,7 +49,9 @@ endif
 govet:
 ifneq ($(HAS_SERVER),)
 	@echo Running govet
-	@$(GO) vet $$(go list ./server/...) || exit 1
+	$(GO) get golang.org/x/tools/go/analysis/passes/shadow/cmd/shadow
+	$(GO) vet $(GOFLAGS) $$(go list ./server/...) || exit 1
```

Comment:
I realized now that the `exit 1` here is actually redundant -- can we strip it out? The `Makefile` will automatically terminate the target if any of the individual steps fail.

Reply:
remove for both? or just in the line 53? @lieut-data

Reply:
done @lieut-data